hydra-access-controls/app/models/hydra/access_control.rb

```diff
@@ -0,0 +1,77 @@
+module Hydra
+  class AccessControl < ActiveFedora::Base
+
+    before_destroy do |obj|
+      contains.destroy_all
+    end
+
+    is_a_container class_name: 'Hydra::AccessControls::Permission'
+    accepts_nested_attributes_for :contains, allow_destroy: true
+
+    attr_accessor :owner
+
+    def permissions
+      relationship
+    end
+
+    def permissions=(records)
+      relationship.replace(records)
+    end
+
+    def permissions_attributes=(attribute_list)
+      raise ArgumentError unless attribute_list.is_a? Array
+      attribute_list.each do |attributes|
+        if attributes.key?(:id)
+          obj = relationship.find(attributes[:id])
+          if attributes.key?('_destroy') && %w{1 true}.include?(attributes['_destroy'])
```

Comment:
Can we use  https://github.com/projecthydra/active_fedora/blob/a58ee090a1564f6098271905b1968ca46565ef36/lib/active_fedora/type/boolean.rb here?